### ISSUE TITLE
How to use a relative path in config.ini for the music ?

### ISSUE THREAD

**Issue by chrixxxx** [hankmorgan/UnderworldExporter#41]
Another small issue I found .

If I don't use an absolute path for the music directory such as this:
UW1_Soundbank=C:\Underworld Exporter\Music Packs\UW1_SB16\ 

but a relative path such as this (with or without the trailing slash):
UW1_Soundbank=\Music Packs\UW1_SB16\ 
or
UW1_Soundbank=Music Packs\UW1_SB16\ 
or
UW1_Soundbank=.\Music Packs\UW1_SB16\

the program isn't apparently able to locate the music, giving an error message about wrong path after you've launched the game (at the bottom of the ultima underworld starting menu, to be clear)

**Comment by hankmorgan** [comment]
I'll see what I can do. The config system needs an overhaul.

**Comment by lookingglassed** [comment]
If that's the path you're using aren't you forgetting to specify the drive, or am I missing something.

**Comment by chrixxxx** [comment]
> If that's the path you're using aren't you forgetting to specify the drive, or am I missing something.

Yes.. if you specify an absolute path you can put the music folder anywhere, as I explained as first thing; but since there's not an installer for UWExporter the possibility to use a relative path addressing when indicating those subdirs would be much more better and it would make everything easily movable though disks and folders - or simply it would allow the renaming of the main folder when an update of UWExporter is released for example - without having to change anything in the config..  while at the moment you have to change the path for subfolders every time you made such modifications.
